src/operator/windowToggle.ts

```diff
@@ -131,10 +131,16 @@ class WindowToggleSubscriber<T, O> extends OuterSubscriber<T, any> {
         const context = { window, subscription };
         this.contexts.push(context);
         const innerSubscription = subscribeToResult(this, closingNotifier, context);
-        (<any> innerSubscription).context = context;
-        subscription.add(innerSubscription);
+
+        if (!innerSubscription.isUnsubscribed) {
```

Comment:
Can we invert this `if` block so it checks `if(innerSubscription.isUnsubscribed) {`?